src/app/controllers/users/edit.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
 flnd.userEdit = {
-    retrieveUser: function(url, $scope, messenger, authService) {
+    retrieveRoles: function($scope, roleProxy, $log, messenger) {
```

Comment:
We need to retrieve the list of available roles in order for the user to be able to select from that list...